frontend/packages/integration-tests-cypress/support/a11y.ts

```diff
@@ -41,6 +41,7 @@ Cypress.Commands.add('testA11y', (target: string, selector?: string) => {
       { id: 'color-contrast', enabled: false }, // seem to be somewhat inaccurate and has difficulty always picking up the correct colors, tons of open issues for it on axe-core
       { id: 'focusable-content', enabled: false }, // recently updated and need to give the PF team time to fix issues before enabling
       { id: 'scrollable-region-focusable', enabled: false }, // recently updated and need to give the PF team time to fix issues before enabling
+      { id: 'list', enabled: false }, // introduced in Cypress 6.0. fix devconsole's sidebar NavList UL to only have LI as direct child elements. see https://github.com/openshift/console/pull/7446
```

Comment:
I'm a little unclear how the linked PR relates to the comment

Reply:
Good catch!  wrong link, I'll update to: https://bugzilla.redhat.com/show_bug.cgi?id=1908772

Reply:
fixed, updated